spec/models/route_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Route, type: :model do
+  describe "validations" do
+    let!(:route) { FactoryBot.create(:route) }
```

Comment:
do we need to use `let!` here?

Reply:
Yes bhai, Unless it will occur error in matching uniqueness of `route_name`.